compiler/tflchef/core/src/ModelChef.cpp

```diff
@@ -755,17 +756,15 @@ void ModelChef::cook(const ::tflchef::ModelRecipe &model_recipe)
     _buffer_vec.emplace_back(buffer_builder.Finish());
   }
 
-  // symbol_tables stores symbol_table of each sub graph
-  // this is used to find tensor ID(index) with tensor name
-  std::vector<std::map<std::string, int32_t>> symbol_tables;
-
   //
   // Create Main graph
   //
 
   _graph_name = "main";
-  auto table = cook_graph<::tflchef::ModelRecipe>(model_recipe);
-  symbol_tables.push_back(table);
+  // Tensor Name -> Tensor ID mapping (per Graph)
+  std::map<std::string, int32_t> symbol_table;
+  cook_graph<::tflchef::ModelRecipe>(model_recipe, symbol_table);
+  _symbol_tables.push_back(symbol_table);
```

Comment:
`std::map` will be copied whenever `push_back` is called. I think there are two ways to avoid copy.

1. Save `std::unique_ptr<std::map<...>>` in `_symbol_tables`. In this case `cook_graph` may return unique_ptr of symbol_table.

2. First create map, and pass the map's reference to `cook_graph`.

```suggestion
  _symbol_tables.emplace_back();
  cook_graph<::tflchef::ModelRecipe>(model_recipe, _symbol_tables.back());
```

Reply:
Looks more complicated to me. I want to go as-is.